### PR TITLE
Build optimization

### PR DESCRIPTION
**Pull request by garethfuller**
# Description

This PR includes a number of adjustments to optimise production builds. Please see comments for specific changes.

Result seems to have got builds from 9mins down to 4mins.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] My changes generate no new console warnings
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,12 +66,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.8",
+    "@babel/plugin-proposal-numeric-separator": "^7.14.5",
     "@sentry/webpack-plugin": "^1.15.1",
-    "@storybook/addon-actions": "^6.3.7",
-    "@storybook/addon-console": "^1.2.3",
-    "@storybook/addon-essentials": "^6.3.7",
-    "@storybook/addon-links": "^6.3.7",
-    "@storybook/vue3": "^6.3.7",
```
Comment (garethfuller) [inline package.json:79]: Removing Storybook for now. We barely use it and it causes a lot of bloat. Also causes a lot of version conflicts. Lets see if we can go without it for now and maybe we can re-introduce if we break the component library away from the main app.

File: package.json [hunk 2]
```diff
@@ -95,17 +86,20 @@
...
     "babel-loader": "^8.2.2",
     "eslint": "^6.7.2",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-vue": "^7.16.0",
     "husky": "^6.0.0",
     "nock": "^13.1.1",
+    "postcss": "^7.0.39",
     "prettier": "^1.19.1",
     "tailwind-config-viewer": "^1.5.0",
+    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.2.17",
```
Comment (garethfuller) [inline package.json:99]: Tailwind dependencies should be in dev dependencies.

File: package.json [hunk 3]
```diff
@@ -53,9 +51,6 @@
     "numeral": "^2.0.4",
     "paraswap": "^4.18.0",
     "parse-filepath": "1.0.2",
-    "postcss": "^7.0.35",
-    "postcss-nesting": "^7.0.1",
```
Comment (garethfuller) [inline package.json:57]: Removing this since it was initially added to support conversion from SASS.

File: .browserslistrc [hunk 4]
```diff
@@ -1,3 +1,3 @@
-> 1%
-last 2 versions
-not dead
+defaults
+not IE 11
+maintained node versions
```
Comment (garethfuller) [inline .browserslistrc:3]: Recommended config from [browserslist readme](https://github.com/browserslist/browserslist#browserslist-).

**Review by 1saf — APPROVED** [review]

**Pull request merged**